packages/virtual-keyboard/src/virtual-keyboard/keyboard-tray.css

```diff
@@ -64,6 +63,8 @@ Tailwind base is purposely not included because we want all styles to be prefixe
         @apply border-b-black border-b;
         @apply hover:bg-slate-300 active:bg-slate-400;
         @apply w-12 h-[1.5rem] flex items-center justify-center;
+        background-color: white;
```

Comment:
This is fine. You can also do `bg-white` inside of an `@apply` block to do it tailwind style.

Reply:
I was wondering about the `@apply`, but figured you'd let me know!

Reply:
Is there a `border-width` `@apply` version I should use as well?

Reply:
https://tailwindcss.com/docs/border-style